Makefile

```diff
@@ -168,7 +168,7 @@ test: clean test.quality test.unit test.migrations_missing test.js test.browser
 	@echo "\nAll tests OK!\n"
 
 test.one: clean
-	$(HONCHO_MANAGE_TESTS) test $(RUN_ARGS)
+	python3 manage.py test $(RUN_ARGS)
```

Comment:
What this change is for, @kewne?

Reply:
A mistake, I changed this to run tests locally then forgot to revert it.